.github/workflows/ci-tests.yml

```diff
@@ -40,8 +40,8 @@ jobs:
             version: py311
             posargs: "--cov-report=xml --cov"
             post-command: codecov
-          - version: py311
-            os: macos-latest
+          - os: macos-latest
+            version: py311
```

Comment:
What's the significance of swapping this? If I understand [the docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) correctly, it works the same either way.

Reply:
No significance other than aligning the order with the previous step. 

I'm just being anal.

Reply:
That's totally fine! Just wanted to make sure I wasn't missing something.